langchain/tracing/__init__.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from langchain.tracing.base import BaseTracer
+from langchain.tracing.nested_json import NestedJsonTracer
+from langchain.tracing.noop import NoOpTracer
+from langchain.tracing.stdout import StdOutTracer
+
+
+def get_tracer() -> BaseTracer:
```

Comment:
I don't have strong opinions about this, this is just the way I chose to implement it atm.

I do think calling a function is a little bit cleaner than having to set a global variable, plus being able to switch tracing implementations by swapping environment variables rather than making code changes is nice too

Reply:
maybe then we should do something at the top of the file like `tracer = get_tracer()` and then use that everywhere (this also has perfect parralels we do with logging, eg `logger = logging.getLogger()`)

thinking about the parallel with logging, you set logging with code rather than env vars, eg `logging.basicConfig(format=FORMAT)` 

i guess the argument here would be that there is less configuration complexity needed so its okay to manage it with env vars? still, i think im partial to letting users set it in code - that also makes it possible for users to write their own logging functionality which may be important as we are open source. 

if you think the convienence of global vars is really that high, we could also do both! eg if the user does not set it explicitly, we then check for the global var

Reply:
> if you think the convienence of global vars is really that high, we could also do both! eg if the user does not set it explicitly, we then check for the global var

By global var, do mean environment var?

Reply:
oops ya